omod/src/main/resources/config.xml

```diff
@@ -40,12 +40,27 @@
 	</advice>
 	<!-- /AOP -->
 
+    <globalProperty>
+        <property>patientmatching.configDirectory</property>
+        <defaultValue>patientmatching</defaultValue>
+        <description>The location of the patient matching configuration folder, can be absolute or relative path
+        </description>
+    </globalProperty>
+
 	<globalProperty>
-		<property>patientmatching.linkConfigFile</property>
-		<description>This file can be found at doc/link_config.xml
+		<property>patientmatching.configFile</property>
+        <defaultValue>patientmatching/config.xml</defaultValue>
```

Comment:
Would this path separator work on windows?

Reply:
This is just a default value they can change it to whatever they want